deploy/rbd/kubernetes/csi-provisioner-rbac.yaml

```diff
@@ -24,6 +24,9 @@ rules:
   - apiGroups: [""]
     resources: ["events"]
     verbs: ["list", "watch", "create", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["endpoints"]
```

Comment:
why does provisioner need endpoints?

Reply:
I retest with deleting this and confirmed that this isn't necessary, so I removed it.
I suspect that I enabled  leader election in the previous tests.